articles/quickstart.md

```diff
@@ -96,26 +108,24 @@ Our goal is to prepare two qubits in a [Bell state](https://en.wikipedia.org/wik
     }
     ```
 
-    This operation may now be called to set a qubit to a known state (`Zero` or `One`).
+    This operation may now be called to set a qubit to a classical state, either returning `Zero` 100% of the time or returning `One` 100% of the time.  `Zero` and `One` are constants that represent the only two possible results of a measurement of a qubit.
 
-    We measure the qubit, if it's in the state we want, we leave it alone, otherwise, we flip it with the `X` gate.
+    The operation `Set` measures the qubit, if it's in the state we want, we leave it alone, otherwise, by executing the `X` operation, we change the qubit state to the desired state.
 
 ### About Q# operations
 
-An operation is the basic unit of quantum execution in Q#. It is roughly equivalent to a function in C or C++ or Python, or a static method in C# or Java.
+A Q# operation is a quantum subroutine. That is, it is a callable routine that contains quantum operations.
```

Comment:
I might suggest another word than "contains" here, perhaps it might be more clear to say that a Q# operation can allocate qubits and call other Q# operations, including the intrinsic operations such as `X` and `M` introduced above?

Reply:
For a new learner, "contains" is better, I think, than the clearer definition you propose.  Also, this is verbatim from the definition of Q# operation in the language doc.